### ISSUE TITLE
Filtering queries by whether Set containing a member does not work

### ISSUE THREAD

**Issue by MarSoft** [pynamodb/PynamoDB#161]
I have a table like this:

```
class Commit(Model):
    project = UnicodeAttribute(hash_key=True)
    commit_sha = UnicodeAttribute(range_key=True)
    branches = UnicodeSetAttribute()
```

Now, I want to retrieve all `Commit`s for certain project which are belonging to certain branch. I use this query:

```
Commit.query(project='awesome', branches__contains='topic')
```

But it doesn't work. I get an error from DynamoDB: `One or more parameter values were invalid: ComparisonOperator CONTAINS is not valid for SS AttributeValue type`
After some digging, I found that `Model._build_filters()` method for value takes the same type as was used for the attribute being filtered, in this case it is `StringSet`. But it is not correct. For `CONTAINS` and `NOT_CONTAINS` checks on sets, we should use corresponding **singular** types.

**Comment by warmax** [comment]
I have the same problem. Is there a workaround?

**Comment by jpinner-lyft** [comment]
@MarSoft @warmax the new condition expression API released in version 3.2.0 correctly handles this:
`Commit.query('awesome', filter_condition=Commit.branches.contains('topic'))`
